WcaOnRails/app/inputs/money_amount_input.rb

```diff
@@ -30,6 +30,7 @@ def input(wrapper_options)
                                         value: value,
                                         id: input_id,
                                         type: "text",
+                                        name: attribute_name,
```

Comment:
What is this change about?

Reply:
So I was looking for a way to make the selection menu (the radio buttons) for free entry display appear immediately when `Entry fee for guests` on the competition creation page is set to zero. 

The `Entry fee for guests` text field has two `input` fields in HTML, but only one of them has a `name` attribute for `$('input[name=""]').on('change')` in javascript.  But the one that has the `name` attribute does not update its `value` attribute in real-time (I think it does update in HTML, but it couldn't be detected by javascript immediately). 

However, the second `input` element of the text field _does_ update its `value` in real-time (can be detected by javascript), but it doesn't have a `name` attribute so I found this way to give it a `name`. This is also why the javascript I added is a little inconsistent with the other line of code, because the `name` of this element does not have `competitions[]` in front of it. 

Not sure if this was the best solution though, since this change affects other money_amount_input elements too. What are your thoughts?

Reply:
Yo no hablo jQuery. Will need feedback by other WST members on this.

Reply:
No hablo jQuery tambien!

Reply:
The money input consists of a visible nameless input and a hidden one with `name=competition[guests_entry_fee_lowest_denomination]`. Whenever the visible one changes, the former value is synchronized, and that's the one that gets submitted with the form. Subscribing to the hidden input with `.on('change', ...)` doesn't work because it's value is set programatically and there is no event. What I'd try is modifying this:

https://github.com/thewca/worldcubeassociation.org/blob/fec596a07772f315a85c4dd9f7c688d9daa4709d/WcaOnRails/app/webpacker/javascript/auto-numeric/index.js#L74

To `.val(...).trigger('change');`. With that in place, you should be able to subscribe to the hidden input changes and keep the visible one nameless.

Reply:
Gotya, thank you Jonatan!